pkg/pillar/types/blob.go

```diff
@@ -30,8 +30,8 @@ type BlobStatus struct {
 	HasDownloaderRef bool
 	// HasVerifierRef whether or not we have started a verifier for this blob
 	HasVerifierRef bool
-	// RefCount number of consumers of this blob
-	RefCount uint
+	// RefObjects map/list of consumers of this blob
+	RefObjects map[string]bool
```

Comment:
Why do we need something this complex to track references?

Reply:
Modified it to RefCount.